packages/dappmanager/test/integration/migrations/removeDnsFromCompose.test.int.ts

```diff
@@ -0,0 +1,113 @@
+import { expect } from "chai";
+import fs from "fs";
+import { removeDnsFromComposeFiles } from "../../../src/modules/migrations/removeDnsFromComposeFiles.js";
+import { params } from "@dappnode/params";
+import { shellSafe } from "../../testUtils.js";
+
+describe("Remove DNS from all compose files", () => {
+    const dncoreNetwork = params.DNP_PRIVATE_NETWORK_NAME;
+    const containerName = "DAppNodeCore-dappmanager.dnp.dappnode.eth";
+    const serviceName = "dappmanager";
+    const randomImage = "chentex/random-logger";
+    const composeDnsRemovalTestPath = process.cwd() + "/test/integration/composeDnsRemoval";
+
+    const composeAlreadyMigrated = `
+version: '3.5'
+networks:
+  ${dncoreNetwork}:
+    name: ${dncoreNetwork}
+    external: true
+services:
+  ${serviceName}:
+    image: ${randomImage}
+    container_name: ${containerName}
+    restart: always
+    networks:
+    ${dncoreNetwork}:
+        ipv4_address: 172.33.1.7
+        aliases:
+          - ${serviceName}.test-migration.dappnode
+          - ${serviceName}.dappnode`;
+
+    const composeToBeMigratedBefore = `
+version: '3.4'
+networks:
+  ${dncoreNetwork}:
+    name: ${dncoreNetwork}
+    external: true
+services:
+  ${serviceName}:
+    image: "${randomImage}"
+    container_name: ${containerName}
+    restart: always
+    dns: 172.33.1.2
+    networks:
+      ${dncoreNetwork}:
+        ipv4_address: 172.33.1.7
+        aliases:
+        - ${serviceName}.test-migration.dappnode
+        - ${serviceName}.dappnode`;
+
+    before("Run random container", async () => {
+        // Create compose
+        await shellSafe(`mkdir ${composeDnsRemovalTestPath}/test-dns-removal`);
+        // Compose to be migrated
+        fs.writeFileSync(
+            `${composeDnsRemovalTestPath}/test-dns-removal/docker-compose.yml`,
+            composeToBeMigratedBefore
+        );
+        // Compose already migrated
+        fs.writeFileSync(
+            `${composeDnsRemovalTestPath}/test-dns-removal/docker-compose-migrated.yml`,
+            composeAlreadyMigrated
+        );
+        // Redeclare global variables
+        params.DNCORE_DIR = composeDnsRemovalTestPath;
```

Comment:
You should not have to do this, the env `TEST=true` should change the dirs already allowing you to test propertly

Reply:
I did not do this, it was already done in the compose migration test